pyproject.toml

```diff
@@ -15,8 +15,7 @@ classifiers = [
   "Topic :: Internet :: WWW/HTTP :: Dynamic Content",
   "Environment :: Web Environment",
   "License :: OSI Approved :: Apache License, Version 2.0",
-  "Programming Language :: Python :: 3.8",
-  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.11",
   "Framework :: Django :: 3.2",
```

Comment:
Could you change this line, too. We are using Django 4.2.

Reply:
I made the changes directly in the github UI